cico_pr_test.sh

```diff
@@ -2,4 +2,11 @@
 
 set -ex
 
+yum -y update
```

Comment:
@budhrg To increase the time out why these packages are being installed ?

Reply:
@amitkrout See #60. The commit should refer to that issue